### PR TITLE
Add documentation for ArchiveRollFiles utility

### PR DESCRIPTION
**Pull request by chroniclekevinpowe**
Provide documentation for ArchiveRollFiles Enterprise feature.

### CODE REVIEW

File: docs/antora/modules/configuration/pages/roll-cycle.adoc [hunk 1]
```diff
@@ -153,7 +153,86 @@ SingleChronicleQueue queue = ChronicleQueue.singleBuilder("/timezone")
...
+ -r <roll-cycles-to-keep>   Number of roll cycles to keep when archiving. Either this or -m must be provided.
+ -vc <value-class>          Name of value class (if queue being archived is backing a QBM)
+----
+
+==== Command-Line Parameters
+
+|===
+| Parameter | Detail | Required?
+
+| -qc | Full or relative path to file to load configuration from for queue being archived. | Y
```
Comment (JerryShea) [inline docs/antora/modules/configuration/pages/roll-cycle.adoc:188]: why is this different to above section?

File: docs/antora/modules/configuration/pages/roll-cycle.adoc [hunk 2]
```diff
@@ -153,7 +153,86 @@ SingleChronicleQueue queue = ChronicleQueue.singleBuilder("/timezone")
...
+----
+$ mvn exec:java -Dexec.mainClass="software.chronicle.enterprise.queue.util.ArchiveRollFilesMain"
+-Dexec.args="-qc config/queue-config.yaml -qn queue1 -m 2024-11-01T12:15:00 -aa DELETE"
+----
+
+Invoking `ArchiveRollFiles` to archive roll files older than the age specified for a `QueueBackedMap` queue.
+[source, shell script]
+----
+$  mvn exec:java -Dexec.mainClass="software.chronicle.enterprise.queue.util.ArchiveRollFilesMain"
+-Dexec.args="-qc /tmp/qbm-replication.yaml -qn queue1 -m 1707289226000 -aa ARCHIVE
```
Comment (JerryShea) [inline docs/antora/modules/configuration/pages/roll-cycle.adoc]: pls use nicely formatted date time

Comment (chroniclekevinpowe) [reply docs/antora/modules/configuration/pages/roll-cycle.adoc]: Intent was to provide an example of `-m` with both conventions - so stick to formatted date time for both examples?

Comment (JerryShea) [reply docs/antora/modules/configuration/pages/roll-cycle.adoc]: I honestly can't see why anyone would use the milliseconds option. You have documented above that it is possible

**Review by JerryShea — APPROVED** [review]

File: docs/antora/modules/configuration/pages/roll-cycle.adoc (hunk 2 above)
Comment (chroniclekevinpowe) [reply docs/antora/modules/configuration/pages/roll-cycle.adoc]: Understood - I've removed it from the documentation.

**Pull request merged**